### PR TITLE
Fixing bad docstrings in conditionals

### PR DESCRIPTION
**Pull request by thevincentadam**
quick fix to update dosctrings

### CODE REVIEW

File: gpflow/conditionals.py [hunk 1]
```diff
@@ -101,7 +103,7 @@ def _conditional(Xnew, X, kern, f, *, full_cov=False, q_sqrt=None, white=False,
     :param X: data points, size M x D.
     :param kern: GPflow kernel.
     :param f: data matrix, M x R, representing the function values at X,
-        for K functions.
+        for R functions.
```
Comment (awav) [inline gpflow/conditionals.py]: Where is R definition? Do we have it in glossary?

Comment (awav) [inline gpflow/conditionals.py]: Where is R definition? Do we have it in glossary?

Comment (thevincentadam) [reply gpflow/conditionals.py]: Just above : 
`:param f: data matrix, M x R,...`
definitely no K!

Comment (thevincentadam) [reply gpflow/conditionals.py]: It is sort of the implicit letter for the usual `num_latent`

**Pull request merged**